src/models.py

```diff
@@ -0,0 +1,47 @@
+import torch
+from torch.nn import Linear
+import torch.nn.functional as F
+from torch_geometric.nn import GCNConv, TopKPooling, global_mean_pool
+from torch_geometric.nn import global_mean_pool as gap, global_max_pool as gmp
+
```

Comment:
We should probably run `isort` so that imports are ordered. (`pip install isort` for it)